tests/test_authentication.py

```diff
@@ -140,13 +139,17 @@ def test_unknown_creds(self):
                               '^access denied: CredentialsLookupError: ')
 
     def test_alternative_credential_lookup(self):
+        # Temporarily use alternate auth
+        old_auth = self.auth
+        self.auth = AlternateAuthentication()
+
         sender = self._sender(credentials=ALTERNATIVE_CREDS)
         req = self._request(sender)
-        lookup_path = "%s.alternative_lookup" % __name__
-        with self.settings(HAWK_CREDENTIALS_LOOKUP=lookup_path):
```

Comment:
I know that's true of a normal `import` statement, but I haven't looked at Django's `import_string()` utility to see what else they do (and thus what other overhead is incurred). (Also, I twitch a little at call-time imports in general. Personal preference.)

But, it's your project, so I'll move stuff around and put that test back (and move the subclass one to a separate test method). Hopefully tonight sometime.